otter/convergence/steps.py

```diff
@@ -575,6 +580,8 @@ def _rcv3_check_bulk_add(attempted_pairs, result):
         next_step = BulkAddToRCv3(lb_node_pairs=to_retry)
         return next_step.as_effect()
     else:
+        # It's unclear when this condition is reached. Should we really be
+        # returning SUCCESS?
```

Comment:
Okay, now that I read the code closer, it definitely seems strange to treat it as a SUCCESS -- this is in the 409 case only.  We should perhaps even consider it a FAILURE.